programs/lazy-distributor/src/lib.rs

```diff
@@ -11,6 +11,15 @@ pub mod token_metadata;
 pub use instructions::*;
 pub use state::*;
 
+#[derive(Clone)]
+pub struct LazyDistributor;
+
+impl anchor_lang::Id for LazyDistributor {
```

Comment:
Shouldn't be needed, anchor exports this doesn't it?

Reply:
Anchor doesn't export it, this code block is in all our programs that are CPI'd into